oidc-api/src/test/java/uk/gov/di/authentication/oidc/entity/LogoutRequestTest.java

```diff
@@ -65,12 +68,13 @@ class LogoutRequestTest {
             URI.create("https://di-authentication-frontend.london.cloudapps.digital/signed-out");
     private static final URI CLIENT_LOGOUT_URI = URI.create("http://localhost/logout");
     private SignedJWT signedIDToken;
-    private static final Subject SUBJECT = new Subject();
+    private static final Subject INTERNL_COMMON_SUBJECT_ID = new Subject();
     private static final String EMAIL = "joe.bloggs@test.com";
     private final ClientRegistry clientRegistry = createClientRegistry();
     private String idTokenHint;
     private String rpPairwiseId;
     private uk.gov.di.orchestration.shared.entity.Session session;
```

Comment:
Not your code, but can this be changed so the type is just Session instead of the full path to the module?